keysign/KeyPresent.py

```diff
@@ -83,7 +82,7 @@ def __new__(cls, *args, **kwargs):
         w.__class__ = cls
         return w
 
-    def __init__(self, key, qrcodedata=None, builder=None):
+    def __init__(self, key, code, qrcodedata=None, builder=None):
```

Comment:
what happens to this `code` variable?
It's not used in this function, is it?
Why is that there?
Even if it was used, what does it represent? It's an overly generic name. Can you either name it more succinctly or add to the docstring? (or both..)

Reply:
I changed the `code` variable name to `discovery_code` and added a docstring.

This variable is used here https://github.com/gnome-keysign/gnome-keysign/blob/d8e3163b0342bad288fa069bb41074bff8d33a20/keysign/KeyPresent.py#L113 and I introduced it with the commit f5fd0d1

This was added to accommodate the asynchronous magic wormhole code. Now KeyPresent requires a `discovery_code` that is the code displayed to the user (now can be the key's fingerprint or the wormhole code) and the `qrcodedata` that is the String that will be encoded in the QR.